src/content/docs/en/anywhere/tegra.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "NVIDIA Tegra (Anywhere)"
+description: "NVIDIA Tegra devices in the Ultramarine Anywhere Initiative"
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+<Alert type="info">
+  This device type is still in the **Wishlist Phase**. All information on this
+  page is subject to change.
+</Alert>
+
+Most of the team has a modern Tegra device, most commonly, the Nintendo Switch.
+There is interest in running Ultramarine on these devices, but we haven’t started work on a port yet.
+
+In the meanwhile, we are tracking the progress of the [Switchroot](https://switchroot.org/) project, which aims to bring Linux to the Nintendo Switch and
+other Tegra-based SoCs, such as the Jetson/Xavier/Orin series, the NVIDIA Shield and more.
+
+The boot process for Tegra devices is similar to other ARM-based Android devices, but with slight differences, especially
+for the Nintendo Switch since it uses a completely custom firmware, due to Nintendo's security measures.
+We'll be mostly documenting the process for the Switch, but the same principles apply to other Tegra devices.
+
+## Nintendo Switch
+
+The Switch loads from a 32GB (or 64GB if you have an OLED version) eMMC storage, which is a detachable module.
+
+Older revisions of the Switch with the X1 SoC (the original 2017 launch model, AKA the "White Box" or "Erista" SoC)
+are the most compatible with Linux, since the hardware support for the X1 itself has been mainlined in the Linux kernel
+and are vulnerable to [CVE-2018-6242 (Fusée Gelée)](https://nvd.nist.gov/vuln/detail/cve-2018-6242) which allows for arbitrary code execution in the boot ROM.
+
+Newer revisions of the Switch with the X1+ SoC (the 2019 revision, AKA the "Red Box" or "Mariko" SoC) and later models
+(i.e the Switch Lite and OLED models, also codenamed "Aula"), are not vulnerable to Fusée Gelée, and require a different exploit which is a combination
+of eMMC interception and a reset glitch to load into a custom bootloader. This requires hardware modifications and is not as straightforward as the original exploit.
+
+"Mariko" SoCs are also not yet supported by the mainline Linux kernel, and require a custom kernel to boot.
+
+The most common bootloader payload used to run arbitrary code on the Switch is [hekate](https://github.com/ctcaer/hekate),
+which is a custom bootloader that provides a graphical boot menu and a partition manager. It can also be used to dump the eMMC storage data and
```

Comment:
```suggestion
a custom bootloader that provides a graphical boot menu and partition manager. It can also be used to dump the eMMC storage data and
```